app/models/patient_phone_number.rb

```diff
@@ -7,4 +7,12 @@ class PatientPhoneNumber < ApplicationRecord
 
   validates :device_created_at, presence: true
   validates :device_updated_at, presence: true
+
+  def self.first
+    order(device_created_at: :asc).first
+  end
+
+  def self.last
+    order(device_created_at: :asc).last
```

Comment:
`order(device_created_at: :desc).first`

Reply:
Having a method `last` call `first` was confusing to me, so made it match.